### PR TITLE
changed vector base to allow backward leg movement

### PR DESCRIPTION
**Pull request by turcottep**
y movement is now positive towards the ground

### CODE REVIEW

**Comment by turcottep** [comment]
@alexandrelafleur how do you add the do not merge tag

File: main.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ def main_loop(self):
 
         self.gui.root.after(self.delay, self.main_loop)
 
+
```
Comment (alexandrelafleur) [inline main.py:52]: extra space

File: positions.py [hunk 2]
```diff
@@ -3,42 +3,42 @@
 
 def get_positions_from_walk_sequence():
     """
-     @return: array of delta positions of 2 joints for each of the 4 feet from walk sequence
-     """
+    @return: array of delta positions ofxstep 2 joints for each of the 4 feet from walk sequence
```
Comment (alexandrelafleur) [inline positions.py:6]: missing space

File: positions.py [hunk 3]
```diff
@@ -96,7 +96,7 @@ def get_angles_from_positions(steps):
     for step in steps:
         angle_step = []
         for leg in step:
-            angle0, angle1 = inverse_kinematic(leg[0], leg[1], 70, 100)
+            angle0, angle1 = inverse_kinematic(leg[1], leg[0], 70, 100)
```
Comment (alexandrelafleur) [inline positions.py:99]: Add comment to explain what you are doing please

File: positions.py [hunk 4]
```diff
@@ -3,42 +3,42 @@
...
-    delta_steps.append([[0, 0], [0, -y_up], [0, 0], [0, 0]])
-    delta_steps.append([[-xstep_front, 0], [-xstep_front, 0], [-xstep_back, 0], [-xstep_back, 0]])
+    yinit = 150
+    xstep = 60
+    y_up = -12.5
+    delta_wheelie = -20
+    x_init = -xstep
+    x_offset = 25
+
+    #initial positions
```
Comment (alexandrelafleur) [inline positions.py]: missing space after #

**Review by alexandrelafleur — CHANGES_REQUESTED** [review]

**Review by alexandrelafleur — APPROVED** [review]

**Pull request merged**